src/mapogcsld.c

```diff
@@ -5383,6 +5383,49 @@ char *msSLDGetFilter(classObj *psClass, const char *pszWfsFilter) {
     } else if (psClass->expression.type == MS_EXPRESSION) {
       pszFilter =
           msSLDParseLogicalExpression(psClass->expression.string, pszWfsFilter);
+    } else if (psClass->expression.type == MS_LIST) {
+      if (psClass->layer && psClass->layer->classitem &&
+          psClass->expression.string) {
+
+        char *pszTmp = NULL;
+        char *pszTmpFilters = NULL;
+        char *token = strtok((char *)psClass->expression.string, ",");
```

Comment:
strtok() is not thread-safe.  The alternative would be strok_r() on Unix and strtok_s() on Windows. But better use msStringTokenize()

Reply:
Thanks for the review. Updated to use `msStringTokenize` (although it is only used in one other function in MapServer). Would `msStringSplit` also suffice? List expressions "do not support quote escaping, or attribute values that contain a comma in them".

Reply:
> Would `msStringSplit` also suffice?

yes, probably